packages/namekit-react/tailwind.config.ts

```diff
@@ -11,6 +11,7 @@ const config: Config = {
       },
     },
   },
+  plugins: [require("@tailwindcss/forms")({ strategy: "class" })],
```

Comment:
A few questions here:

1. Given how you've moved this dependency into `namekit-react`, should we update this documentation in the README.md file in the root of our monorepo? https://github.com/namehash/namekit/blob/85c83708b058ea383457faba1001a1de309967bf/README.md?plain=1#L518-L522 For example, exactly who needs to follow this instruction? Is it anyone building an app that adds namekit-react as a dependency? Or does this instruction only need to be followed if you import specific UI components from namekit-react? Etc.. Appreciate if you could write out a lot of details about this to make it more clear. I don't understand this currently.
2. Can you help me understand more specifically why we need this plugin at all? I don't mind adding a plugin if it only impacts code directly in our own package. What I don't like is when we do things that cause unstandard complexity for everyone else who might use our package. For example, could we effectively replace the use of this plugin by just manually adding more classnames to the related form UI components?

Reply:
1. This instruction only needs to be followed if the user imports our `Input` component from `namekit-react`. Other components, like our checkbox, are already compatible with the `@tailwindcss/forms` reset, so no additional configuration is needed for them. However, if the user imports the `Input` component and applies a global CSS reset (such as `@tailwindcss/forms` or similar), it may cause unintended styling issues with the `Input` component.
<br>To avoid these issues, users who import our `Input` component should follow the instructions to configure `@tailwindcss/forms` with the class strategy, or alternatively, we could add the reset class directly to the `Input` component to ensure it displays correctly in any setup.

2. The `@tailwindcss/forms` plugin is basically a CSS reset focused on form elements. Browsers come with their own default styles for things like checkboxes, inputs, and textareas, which can make it tough to get a consistent look with Tailwind. This plugin wipes those defaults, making it easier to style form components the way we want.
<br>We added `@tailwindcss/forms` to `namekit-react` because our checkbox component already depends on that reset to look right. But our `Input` component doesn’t account for this reset yet. So, if someone using our package applies a global CSS reset on form elements, it can make our `Input` look weird.
<br>One way to handle this and get rid of the warning is to add the `@tailwindcss/forms` reset class directly to our `Input` component. That way, even if the user has a global CSS reset, our `Input` won’t be affected.
<br>This isn’t just a `@tailwindcss/forms` issue, it could happen with any CSS reset. We just noticed it specifically with `@tailwindcss/forms` because we were using it globally on Nameguard, which made the `Input` component look weird in forms there.


To wrap things up, this is a very specific issue, it only occurs if the user imports the `Input` component from `namekit-react` and has a CSS reset applied globally to form elements. Given how specific this is, I believe we have three options:

A: Remove the comment and let the user figure it out independently if that rare scenario happens.
B: Add the `@tailwindcss/forms` reset to our `Input` component.
C: Include a more generic warning, such as:
"This package includes custom styles that may be affected by other CSS resets or frameworks (e.g., Tailwind CSS forms). If you encounter unexpected styling issues, consider checking for conflicts with any global or opinionated CSS resets in your project."

Reply:
That's what happens with our contact form, for example
<img width="1247" alt="image" src="https://github.com/user-attachments/assets/a01be7a6-f7cb-4993-a670-bdecfe1d0fd8">

Reply:
@lightwalker-eth I’ve added a few reset classes to our Input component and removed the warning about Tailwind forms.

https://github.com/namehash/namekit/pull/453/commits/870fe0bb54a1bfa86b0f2ca596d794831978d3a5